.github/workflows/nightly.yaml

```diff
@@ -55,4 +55,4 @@ jobs:
           GCP_SERVICE_ACCOUNT_KEY: ${{ secrets.GCP_SERVICE_ACCOUNT_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          args: --namespace=test-clusters --init-manifest=cilium.yaml --image=cilium/cilium-perf-test:8cfdbfe "/usr/local/bin/run_in_test_cluster.sh" "--prom-name=prom" "--prom-ns=prom" "--duration=30m"
+          args: --namespace=test-clusters --init-manifest=cilium.yaml --image=cilium/cilium-perf-test:8cfdbfe "/usr/local/bin/run_in_test_cluster.sh" "--prom-name=prom" "--prom-ns=prom" "--duration=30m" "--manifest-path=/go/src/github.com/cilium/cilium-perf-test/1.8/manifests/"
```

Comment:
From the docs of `github.com/cilium/cilium-perf-test/1.8` it seems that directory is dedicated for the 1.8 branch, shouldn't this be a different directory?

Reply:
I have a cleanup of this repo in my backlog, there is nothing 1.8 specific in this branch. After I confirm that everything works here, I will do the cleanup.